oneargopy/Argo.py

```diff
@@ -227,54 +345,432 @@ def __load_sprof_dataframe(self) -> pd:
             # with pd.option_context('future.no_silent_downcasting', True):
         result_df = expanded_df.pivot(index='index', columns='parameter', values='data_type').fillna(0).astype('int8')
 
-        # Fill in parameters and dacs before removing rows
-        # Fill in source_settings information based off of synthetic file
+        # Fill in source_settings information based off of synthetic file before removing rows
         if self.download_settings.verbose: print(f'Filling in source settings information...')
         self.source_settings.set_avail_vars(sprof_index )
```

Comment:
it looks like you added blanks everywhere after sprof_index; please remove them all